common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -709,6 +709,13 @@ object CometConf extends ShimCometConf {
     .booleanConf
     .createWithDefault(sys.env.getOrElse("ENABLE_COMET_STRICT_TESTING", "false").toBoolean)
 
+  val COMET_TEST_SHOW_ANSWERS: ConfigEntry[Boolean] = conf(s"$COMET_PREFIX.testing.showAnswers")
```

Comment:
```suggestion
  val COMET_TESTING_SHOW_ANSWERS: ConfigEntry[Boolean] = conf(s"$COMET_PREFIX.testing.showAnswers")
```
To be consistent with the env var name (line 717)